cmd/service-controller/services.go

```diff
@@ -298,7 +298,8 @@ func serveServices(m *ServiceManager) http.Handler {
 				if err != nil {
 					http.Error(w, err.Error(), http.StatusInternalServerError)
 				} else {
-					event.Recordf("Service %s exposed", options.GetServiceName())
+					message := fmt.Sprintf("Service %s exposed", options.GetServiceName())
```

Comment:
Should an event be generated in the error case above?

Reply:
True, updated in f21636ed7a402bec9cd6af48558a79b2497110a4